device/dell/x86_64-dell_s6000_s1220-r0/Force10-S6000/context_config.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
what is the purpose to add gearbox setting to dell s6000? s6000 is phyless design.

Reply:
Guohan,  The s6000 actually has a sesto PHY and was used in our original prototype of gearbox and Phy abstraction interface.  We have changed the development vehicle to the Silverstone DP as it is using a real Milleneo Broadcom PHY supported by Broadcom Phy Abstraction Interface code.

The s6000 is now only used the S6000 for virtual Switch if that makes sense.   Silverstone DP PR was not approved yet (its been pending for a while) but we did merge it into our SONiC base and are using that for real testing on JR2 / Gearbox.

Reply:
@lguohan

Reply:
I am not aware of s6000 has any phy. which port has the phy?